sda/service.go

```diff
@@ -72,6 +72,7 @@ type serviceFactory struct {
 	translations map[string]ServiceID
 	// Inverse mapping of ServiceId => string
 	inverseTr map[ServiceID]string
+	mutex     sync.Mutex
```

Comment:
Hah - found it:
https://travis-ci.org/dedis/cothority/builds/146393746#L203
There `go` complains about a race-condition on the `serviceFactory`-structure, so I put in a mutex (and created a deadlock...)